metabase/metabase
# Update Metabase banner upsell UI

Closes https://github.com/metabase/metabase/issues/45407

### Description

Update Metabase banner upsell UI to match the design

### How to verify

Run Metabase in OSS edition

1. Sign in as an admin
1. To go a dashboard or a question.
1. Sharing > Embed > Static embed > Look and Feel
1. See that the Metabase banner Upsell UI has been updated to match the [design](https://www.figma.com/design/1YcsFf5yYdFZTguwoKVdkj/Re-design-the-transparent-background-setting-in-interactive-embedds?node-id=917-6518&t=LcLo2AyM8ZhbZt1S-0)

### Demo
![image](https://github.com/metabase/metabase/assets/1937582/f889746e-4239-4e3f-8741-dbb4c4e5f68d)


### Checklist
